src/UI/Buyer/src/app/services/order/checkout.service.ts

```diff
@@ -50,6 +50,11 @@ export class CheckoutService {
     })
   }
 
+  async checkForSellerOwnedProducts(lineItems: HSLineItem[]): Promise<void> {
+    const someSellerProducts = lineItems.some((li) => li.SupplierID === null)
```

Comment:
Nitpick but would you mind changing this to hasSellerProducts the variable someSellerProducts made me think this returned a list of line items that contained seller products.

Reply:
Done.